authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/impl/hotlist/HotlistServiceImpl.java

```diff
@@ -57,27 +57,30 @@ public void unblock(String id, String idType) throws IdAuthenticationBusinessExc
 
 	/**
 	 * Retrieve the Hotlist Status information.
-	 * @param id 			the id_hash
-	 * @param idType 		the id_type
-	 * @return HotlistDTO 	consist of hotlisting information
+	 * 
+	 * @param id     the id_hash
+	 * @param idType the id_type
+	 * @return HotlistDTO consist of hotlisting information
 	 * @throws IdAuthenticationBusinessException
 	 */
 	/*
 	 * (non-Javadoc)
-	 * @see io.mosip.authentication.core.spi.hotlist.service.HotlistService#getHotlistStatus(java.lang.String, java.lang.String)
+	 * 
+	 * @see io.mosip.authentication.core.spi.hotlist.service.HotlistService#
+	 * getHotlistStatus(java.lang.String, java.lang.String)
 	 */
 	@Override
-	public HotlistDTO getHotlistStatus(String id, String idType) throws IdAuthenticationBusinessException {
-		HotlistDTO dto = null;
+	public HotlistDTO getHotlistStatus(String id, String idType) {
```

Comment:
Please return Optional<HotlistDTO>  and when empty, return Optional.empty().

Reply:
getStatus method will return BLOCKED or UNBLOCKED. null wont be returned.